prov/rxm/src/rxm_conn.c

```diff
@@ -1275,14 +1273,17 @@ rxm_conn_handle_event(struct rxm_ep *rxm_ep, struct rxm_msg_eq_entry *entry)
 		}
 		reject_reason = cm_data->reject.reason;
 
-		if (reject_reason == RXM_CMAP_REJECT_GENUINE)
+		if (reject_reason == RXM_CMAP_REJECT_GENUINE) {
 			FI_WARN(&rxm_prov, FI_LOG_FABRIC, "connection reject: "
 			       "(reason: RXM_CMAP_REJECT_GENUINE)\n");
```

Comment:
I know this isn't part of this patch, but this reason explanation (genuine?) could be improved.

Reply:
This is just a regular connection reject from a peer because the peer's unavailable or went into error state. I can update the log to say the same and remove reference to "genuine" since that's something internal to the provider.